zbus/src/abstractions/process.rs

```diff
@@ -1,5 +1,82 @@
+#[cfg(not(feature = "tokio"))]
+use async_process::unix::CommandExt;
+use std::{ffi::OsStr, process::Stdio};
 #[cfg(target_os = "macos")]
-use std::{ffi::OsStr, io::Error, process::Output};
+use std::{io::Error, process::Output};
+
+/// A wrapper around the command API of the underlying async runtime.
+#[allow(unused)]
+pub struct Command(
+    #[cfg(not(feature = "tokio"))] async_process::Command,
+    #[cfg(feature = "tokio")] tokio::process::Command,
+);
+
+impl Command {
+    /// Constructs a new `Command` for launching the program at path `program`.
+    #[allow(unused)]
```

Comment:
It's fine to have unused warnings in the intermediate commits, so please remove all `allow(unused)`.